projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -229,31 +242,6 @@ It's recommended to inject all dependencies as effect function arguments for eas
 
 ## Registering Root Effects
 
-After you've written class-based or functional effects, you must register them so the effects start running. To register root-level effects, add the `EffectsModule.forRoot()` method with an array or sequence of effects classes and/or functional effects dictionaries to your `AppModule`.
```

Comment:
Can you add a note about NgModules similar to https://github.com/sheikalthaf/platform/blob/6f926343e3f50e1d9a35c90b3c76c811d3fc9dbe/projects/ngrx.io/content/guide/router-store/index.md?plain=1#L30 please?